tests/upgrades/test_subscription.py

```diff
@@ -162,36 +159,24 @@ def test_post_subscription_scenario_autoattach(
             1. Pre-upgrade content host should get subscribed.
             2. All the cleanup should be completed successfully.
         """
-        docker_vm = settings.upgrade.docker_vm
-        pre_test_name = dependent_scenario_name
-        docker_container_name = f'{pre_test_name}' + "_docker_client"
-        docker_hostname = execute(
-            docker_execute_command,
-            docker_container_name,
-            'hostname',
-            host=docker_vm,
-        )[docker_vm]
-        org = entities.Organization().search(query={'search': f'name="{pre_test_name}_org"'})[0]
-        request.addfinalizer(org.delete)
-        loc = entities.Location().search(query={'search': f'name="{pre_test_name}_loc"'})[0]
-        request.addfinalizer(loc.delete)
-        act_key = entities.ActivationKey(organization=org).search(
-            query={'search': f'name="{pre_test_name}_ak"'}
+        rhel_contenthost = Broker().from_inventory(
+            filter=f'hostname={pre_upgrade_data.rhel_client}'
         )[0]
-        request.addfinalizer(act_key.delete)
-        host = entities.Host(organization=org).search(
-            query={'search': f'name="{docker_hostname.casefold()}"'}
+        host = target_sat.api.Host().search(query={'search': f'name={rhel_contenthost.hostname}'})[
```

Comment:
This could be simplified to 
```suggestion
        host = rhel_contenthost.nailgun_host
```
However it is the same in the background.

Reply:
@ogajduse I will have to revert your suggestion here as the `host` object here is of type host class, not the content_host which not able to get the nailgun host